src/Sylius/Behat/Client/Request.php

```diff
@@ -179,6 +179,17 @@ public function addSubResource(string $key, array $subResource): void
         $this->content[$key][] = $subResource;
     }
 
+    public function updateSubResource(string $subResource, int $key, array $newValues): void
+    {
+        // @TODO : FIX ME, this doesn't edit the resource but creates a new one
+        $this->content[$subResource][$key] = $newValues;
```

Comment:
I need help with this part. I don't know how to do it.

Reply:
ping @lchrusciel @pamil @Zales0123

Reply:
Hello @lchrusciel ,
No problem, it's not urgent, however, i think we will need this feature several times in the future, so we will have to look into it.

Reply:
Hi @oallain, I've just quickly checked it out and it seems to work properly for me 😱. Again, sorry for the delay, I'll do a deep review tomorrow probably so I hope I'll be able to give you more detailed feedback soon :)

Reply:
I can confirm that it still seems to work :D Can you shed some light on this issue?

Reply:
After discussing it with @Zales0123 on Slack we concluded that a sub-resource shouldn't be edited from the resource.